app/src/main/res/layout/rc_phase_shift_setup.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="5dp">
+
```

Comment:
Yes. I will rename it while solving issue #569 . For now I am using `diode_setup.xml`